examples/tutorial.py

```diff
@@ -108,8 +108,7 @@
             if run == 0:
                 plotting.plot_iodata(
                     x, y, title=task.name, savefig=True,
-                    fname=f'io_{task.name}_{activation}',
-                    fig_dir=OUTPUT_DIR,
+                    fname=os.path.join(OUTPUT_DIR, f'io_{task.name}_{activation}'),
```

Comment:
Somehow I left this and similar other changes in the `tutorial.py` out from my previous pull request. I fixed it now.